akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -38,3 +45,20 @@ Java
 For a full list of strategies see the public methods on `SupervisorStrategy`
 
 
+## Bubbling exceptions up
+
+In some scenarios it may be useful to push the decision about what to do on a failure upwards in the Actor hierarchy
+(in untyped Akka Actors this is how it works by default) and let the parent actor handle what should happen on failures.
+
+// FIXME is it useful that we should introduce something for it perhaps?
+
+// IDEA 1: parent will have to use watch, and act on children terminating
+// problems: no idea what exception happened (maybe that is good, to not tightly couple parent and children)
```

Comment:
That could be the reason for that there is a `def failure: Throwable` in `Terminated`. Check if that isn't populated exactly for child failures.

Reply:
see also "bubbling up failures" in https://github.com/akka/akka-meta/issues/46

Reply:
Thanks, I missed that there actually was something more than a mention that we need to solve it in the meta issue.